Sources/PubNubChat/PubNub/Models/ChatMessage.swift

```diff
@@ -37,25 +37,25 @@ public struct ChatMessage<CustomData: ChatCustomData>: Identifiable, Codable {
   
   public struct Content: JSONCodable {
```

Comment:
Would it be a good idea to rename this class to `Payload`?

Reply:
Lets align with the other platforms on how to describe this concept (whatever that might be).  It'll make it easier for our docs to convey the same information across the each platform.

Reply:
What was the resolution on this: Payload, Content, or other?

Reply:
Android uses `NetworkMessagePayload` because they always put a prefix like `Network` or `DB` before a class name. We don't add any prefixes so I think the resolution should be `MessagePayload` and I will include it in my PR. Then we can create a new `ChatMessage` object in this way:

```
ChatMessage<ModelData>.MessagePayload(
  id: UUID().uuidString,
  text: inputText,
  custom: constructOutgoingContentCustomData(),
  createdAt: Date()
)
```

Or if you don't like a redundant message keyword because of nested types we use (Android doesn't use it), we can rename it to `Payload` and use it like that:

```
ChatMessage<ModelData>.Payload(
  id: UUID().uuidString,
  text: inputText,
  custom: constructOutgoingContentCustomData(),
  createdAt: Date()
)
```